tests/e2e/gc_block_volume_expansion.go

```diff
@@ -145,6 +147,10 @@ var _ = ginkgo.Describe("[csi-guest] Volume Expansion Test", func() {
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		svcClient, svNamespace = getSvcClientAndNamespace()
 		setResourceQuota(svcClient, svNamespace, defaultrqLimit)
+
+		if isGCCSIDeploymentPODdown {
+			_ = updateDeploymentReplica(client, 1, vSphereCSIControllerPodNamePrefix, csiSystemNamespace)
```

Comment:
why are you ignoring the error for `updateDeploymentReplica()`? I've noticed this in couple of places.

Reply:
That returns Deployment itself not error. If Error that is handled in the method itself

Reply:
ok, I got confused with `updateDeploymentReplicawithWait` method